### PR TITLE
usage example for sprite set y

### PR DESCRIPTION
**Pull request by kay-kaushik**
# Description
This pull request features usage exaples for sprite_set_y(). The usage example is clear and concise to help the user understand how to implement the function.

# Features
+ clear and concise code
+ images
+ perfect example

# Changes
5 files have been added. 2 code, 1 text and 1 png and 1 zip files.

# Benefits
These high quality usage examples give more context to the functions. This will help users reading the documentation to better understand the function. This will result in faster development.

# Testing
All the changes have been tested on Kartik Kaushik's system

# Testing Images

s2
s1

### CODE REVIEW

File: src/assets/usage-examples-code/sprites/sprite_set_y/sprite_set_y-9-sprite-y.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+### sprite_set_y() usage example
```
Comment (omckeon) [inline src/assets/usage-examples-code/sprites/sprite_set_y/sprite_set_y-9-sprite-y.txt:1]: Improve this heading to follow the style used in existing usage examples.

File: src/assets/usage-examples-code/sprites/sprite_set_y/sprite_set_y-9-sprite-y.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+The following code is an example of using sprite set y which sets the y position of a sprite on a splashkit window.
```
Comment (omckeon) [inline src/assets/usage-examples-code/sprites/sprite_set_y/sprite_set_y-9-sprite-y.txt:3]: Update this to be a bit clearer and use capital first letters on the function name with a proper link.

File: public/usage-examples-images-gifs/sprites/protSpriteSheet-R.png
Comment (omckeon) [inline public/usage-examples-images-gifs/sprites/protSpriteSheet-R.png:1]: Can you simplify the name of the image? And then make sure to update this PR to use the instructions for including resources.

**Review by omckeon — CHANGES_REQUESTED** [review]
Just need to update a few things mentioned in the comments

**Comment by omckeon** [comment]
Make sure you have run `npm run build` successfully as well. (The check should not be failing.)

File: src/assets/usage-examples-code/sprites/sprite_set_y/sprite_set_y-9-sprite-y.txt (hunk 1 above)
Comment (kay-kaushik) [reply src/assets/usage-examples-code/sprites/sprite_set_y/sprite_set_y-9-sprite-y.txt]: I have changed the heading

File: src/assets/usage-examples-code/sprites/sprite_set_y/sprite_set_y-9-sprite-y.txt (hunk 2 above)
Comment (kay-kaushik) [reply src/assets/usage-examples-code/sprites/sprite_set_y/sprite_set_y-9-sprite-y.txt]: I have done as asked.

File: public/usage-examples-images-gifs/sprites/protSpriteSheet-R.png
Comment (kay-kaushik) [reply public/usage-examples-images-gifs/sprites/protSpriteSheet-R.png:1]: changed the name and added zip resource folder

**Review by NidhishaPahade — COMMENTED** [review]
Everything looks good to me.

**Pull request merged**